src/main/java/bob/Event.java

```diff
@@ -0,0 +1,26 @@
+package bob;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+
+public class Event extends Task {
+    protected LocalDate from;
+    protected LocalDate to;
+
+    public Event(String description, String from, String to) throws DateTimeParseException {
+        super(description);
+        this.from = LocalDate.parse(from, Parser.INPUT_DATE_FORMATTER);
+        this.to = LocalDate.parse(to, Parser.INPUT_DATE_FORMATTER);
+    }
+
+    public String format() {
+        return "event | " + super.format() + " | " + this.from.format(Parser.INPUT_DATE_FORMATTER)
```

Comment:
yea youre right that the naming can be better. the format method is to format the task in the format stored in the hard disk. input format is used so that when loading the tasks from the hard disk, we can load the tasks as if we are reading the tasks in the file.